hubconf.py

```diff
@@ -6,7 +6,10 @@
     import torch
     model = torch.hub.load("Megvii-BaseDetection/YOLOX", "yolox_s")
 """
-dependencies = ["torch"]
+from tools.hub import ensure_installed
+
+HUB_REQUIREMENTS = ["loguru", "numpy", "torchvision", "thop", "onnx", "opencv_python", "tabulate"]
```

Comment:
Would opencv-python-headless do here?

Reply:
Yep, I've specified a version for it and it seems to be working. Without a specified version it was not working in google colab